swarmzero_config_example.toml

```diff
@@ -1,4 +1,7 @@
+max_iterations = 10
```

Comment:
we won't need a top level attribute anymore. the one nested under `model` replaces this